spec-dtslint/operators/pluck-spec.ts

```diff
@@ -30,13 +30,29 @@ it('should support nested object of more than 6 layer depth', () => {
 });
 
 it('should accept existing keys only', () => {
-  const a = of({ name: 'abc' }).pipe(pluck('xyz')); // $ExpectError
+  const a = of({ name: 'abc' }).pipe(pluck('xyz')); // $ExpectType Observable<unknown>
 });
 
 it('should not accept empty parameter', () => {
-  const a = of({ name: 'abc' }).pipe(pluck()); // $ExpectError
+  const a = of({ name: 'abc' }).pipe(pluck()); // $ExpectType<unknown>
```

Comment:
This doesn't look like a valid dtslint type expectation. Should it be `$ExpectType Observable<unknown>`?